monitoring/homepage_test.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-console */
+/* global $http $util $secure */
+/*
+ * Synthetics API Test Documentation:
+ * https://docs.newrelic.com/docs/synthetics/new-relic-synthetics/scripting-monitors/write-synthetics-api-tests
+ */
+const assert = require('assert');
+
+function randomArray(digits, max){
+  return Array.from({length: digits}, () => Math.round(Math.random() * max));
```

Comment:
Can we get a constant seed (maybe per location)?

Reply:
I just removed the randomness; no real need we should just check all of the first 256 of the index especially when it doesn't really cost to do so! :)

Reply:
Maybe the parts of the index that we check should be randomized? we check max 256 the [start + offset] can be random? @davidnuescheler